api/swagger.yaml

```diff
@@ -10443,7 +10443,10 @@ paths:
           type: "string"
           required: true
         - name: "platform"
-          type: "string"
+          type: "array"
+          items:
+            type: "string"
```

Comment:
I'll open a follow-up to update the docs/api/version-history.md - those docs probably still need to be moved to the API module as well.